ext/rotoscope/rotoscope.c

```diff
@@ -215,17 +217,18 @@ static void event_hook(VALUE tpval, void *data)
     return;
 
   rs_tracepoint_t trace = extract_full_tracevals(trace_arg, &trace_path);
-  gzprintf(config->log, RS_CSV_FORMAT, RS_CSV_VALUES(trace));
+  fprintf(config->log, RS_CSV_FORMAT, RS_CSV_VALUES(trace));
 }
 
-static void close_gz_handle(Rotoscope *config)
+static void close_log_handle(Rotoscope *config)
 {
   if (config->log)
   {
     if (!in_fork(config))
     {
-      gzclose(config->log);
+      fclose(config->log);
     }
+    close(fileno(config->log));
```

Comment:
Would this make the `fclose` above unnecessary? I would've assumed that `fclose` would basically be a "flush + close FD + free `FILE*` mem".